java/code/src/com/redhat/rhn/taskomatic/task/ClearLogHistory.java

```diff
@@ -73,7 +73,10 @@ public void execute(JobExecutionContext context)
         now.set(Calendar.MILLISECOND, 0);
         Date limitTime = now.getTime();
 
-        log.info("Clearing log history older than: {}", LocalizationService.getInstance().formatCustomDate(limitTime));
+        if (log.isInfoEnabled()) {
```

Comment:
same as before, but now for info messages

Reply:
Same here for `LocalizationService.getInstance().formatCustomDate(limitTime)`